### DISCUSSION TITLE
Why does the "For you (beta)" tab just show random strangers and repositories I never heard of?

### DISCUSSION THREAD

**Discussion by VorpalBlade** [community/community#28079] [Other Feature Feedback, Questions, & Ideas]
# The problem
I was looking at the new "For you (Beta)" feed tab that showed up. While the idea seems interesting, the feature appears useless in it's current form. It contains events I don't care about, for people I never heard of, regarding repositories I have no idea what they are.

I tried the filter menu, but none of the filters made it any better, just removing different irrelevant subsets. Finally I unchecked everything from the filters and was left with a blank page, which seems to mirror my feelings towards this feature at the moment.

# The solution?
The feature might be useful if the feed contained things relevant to repositories I had stared. Or perhaps people I followed (if I followed anyone, which I don't, nor do I see any point in doing so. I care about projects, not individual strangers on the internet).

Maybe it could look at repositories I filed bugs for or contributed to? I would suggest "things I looked at" but please don't open that can of privacy nightmares.

Either way, this new "For you (Beta)" tab needs to do a way better job than it does currently in finding things relevant to me if it is going to be useful in any way.

**Comment by ghost** [comment]
I did not sign up for this. Thisisillegal!!! I'm only keeping this account open for evidence. Ignoring me is irrelevant.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by VorpalBlade** [reply]
You clearly didn't read my post. I already described that I tried the filter and that everything in it was useless. I also said I unchecked all the filters and was left with an empty page, as you also show in your screenshot. Which makes the whole page useless.

Would you please reply with a non-cut-and-paste answer that actually addresses what I wrote. Because what you just did is condescending and I expect better of GitHub staff.

**Comment by spark9026** [comment]
### Redirect Protocols
I was looking at the [**"https"**](https://spark9026.online) protocol redirect to [**"http"**](http://spark9026.online) protocol. While the idea seems missing in Google Internet Service.  Is this interesting ?, The feature appears bright useless not  get solution. It contains care about website, for people I never heard of, regarding repositories I have no idea what they are.

I tried the filter sea of google, but none of the filters made for this type of problem. Finally I unchecked everything from the filters and was left with a blank page, which seems to mirror my feelings towards this feature at the moment.

**Reply by jfullstackdev** [reply]
can you have the screenshots of that ?

**Comment by spark9026** [comment]
### **NOT FOUND IN SEA OF GOOGLE.**

**Comment by Burnett01** [comment]
The "For you" tab is awesome and I hope they keep it or at least make it available to those who like it.
I found cool projects that way and wouldn't want to miss it.

**Comment by mikekavouras** [comment]
@VorpalBlade if you click the `...` on the top right of a card you should see a bit of context as to why you're seeing an event. It would be helpful data for us if you could share some of the "reasons" associated with events that you don't find relevant. Maybe there's a pattern.

<img width="356" alt="CleanShot 2022-09-08 at 10 53 54@2x" src="https://user-images.githubusercontent.com/627794/189154951-0f3213fa-1da8-4393-a0be-72e93a6e4717.png">

**Reply by VorpalBlade** [reply]
Sure, here are some examples:
![image](https://user-images.githubusercontent.com/3315306/189297844-e7a18675-cb62-4069-abd4-763465b217cd.png)
This claims I starred that repository. I have not:
![image](https://user-images.githubusercontent.com/3315306/189297815-5c94822d-2723-4ea9-9154-47fe2ba3ac4c.png)
So that is just buggy!

![image](https://user-images.githubusercontent.com/3315306/189298156-02e6b976-f0af-4ad5-adff-c6a7b03df7d4.png)
I have no idea who this is, nor what this project is. So I went looking through their other repositories and I found [one](https://github.com/foreachthing/RecyclingSymbols) that I used and filed a bug report with. I would not exactly call that "collaborated with". Besides, just because I happened to use some OpenSCAD files of theirs does not imply I'm interested in their C++ projects. I care about projects, not people. At least until I have had *way* more interactions with them than this.

![image](https://user-images.githubusercontent.com/3315306/189299146-d8f52885-c2ad-47d2-958e-53221ac17cae.png)
Similar story, I have had more interactions with this person (the chezmoi dot file manager). Doesn't mean I care about what they happen to use a "geometry engine" for.

These are the only relevant entries:
![image](https://user-images.githubusercontent.com/3315306/189299468-479ffce4-9bac-4e79-bb2e-c958ca15d885.png)
Here I actually reported bugs and use kiwix. I have no idea who "kelson42" is however. So the reason for it being relevant is completely wrong.
![image](https://user-images.githubusercontent.com/3315306/189300602-dec3464c-f16e-4ce2-905a-e78d87346784.png)
Again, relevant, but for the wrong reasons. I have reported bug to this project. No clue who "lukasmatena" is.


![image](https://user-images.githubusercontent.com/3315306/189300225-48dd6bc6-b13f-40ec-b048-5ad2014fe818.png)
Well, that is just plain wrong. I don't follow anyone. Nor do I plan to. GitHub is not a social network. I care about projects here, not people.

![image](https://user-images.githubusercontent.com/3315306/189300368-79f8ea57-5fd7-4dc5-bbb8-910de0f79d93.png)
Again, no I haven't ever starred this. Nor have I even heard of this.

![image](https://user-images.githubusercontent.com/3315306/189300923-9676d85c-cf6b-41ed-bce1-785ecb643e8e.png)
Well that one is hard to argue with. But no it is not relevant to me. I use KDE, not Cinnamon.

![image](https://user-images.githubusercontent.com/3315306/189301056-e2ce73df-e451-4509-aaca-9d5d73c3c4f0.png)
Who is this? What is this? No idea on either account. Went looking through their repositories. Nothing looked familiar.

![image](https://user-images.githubusercontent.com/3315306/189301316-c07ef7e9-48ce-4459-8c7c-ce93a74b4486.png)
Hm okay, same name here. Msgpack sounds vaguely familiar from a previous work place. I might have reported a bug or something at some point? Not sure. If so there should be a button that says "I don't care about this project any more, it is no longer relevant to me." (Besides the fact that it is actually thinking it is relevant because of some person I don't know.)

![image](https://user-images.githubusercontent.com/3315306/189301744-f575bfea-32fc-49a9-8afd-3d3e895d89c9.png)
Okay, this one I can actually see how it happened: ROS. I used to do ROS stuff. I don't any more. However even back then this would have been irrelevant as I never heard of this mavlink. But the algorithm is imperfect. I can understand that. However: How do I tell the system I moved on from ROS and research robotics?

![image](https://user-images.githubusercontent.com/3315306/189302151-f0606207-c943-4339-abd5-6da6940a90cf.png)
Again, the system seem to focus on people, not projects. I care about their chemoi project. Not some geometry thing.

![image](https://user-images.githubusercontent.com/3315306/189302428-d5b2e14a-0749-4849-9788-91f953509c3e.png)
And back to lying about stars. At least this time I actually do use zsh, so this might be marginally related.

I think (apart from the bugs where it think I starred a project that I didn't), the main problem is that this is people focused instead of project focused. GitHub *is not* a social network. It should not *try to be a* social network. The feature would be far more useful if it was project focused (tell me about new releases for projects I have contributed to or starred) rather than focused about developers I don't know of.

**Reply by VorpalBlade** [reply]
@mikekavouras Hi, it has been almost two weeks since I posted the information you asked about. Yet I don't see any comments from you (or someone else at GitHub) with respect to my reply. I'm especially interested in hearing your take on the fundamental flaw of this feature (that it is people focused instead of project focused).

This kind of radio silence is not a good image for GitHub. It is very easy to interpret it as the company (GitHub) or parent company (Microsoft) having decided on a course of action ("lets try to be a social network for software, yay!") and ignoring the feedback from the community. I hope that is not the case.

**Comment by Ky6uk** [comment]
It might be good to have an option for "collaborators" in the feed filters. Sometime in the feed, I see a lot of unrelated crap with the reason `You're seeing this because you collaborated with $userName`.

**Comment by ZoelefackReynaNk** [comment]
Uhhhhm, please am kind of not in scope!
Will anyone bother explaining me the functionality of GitHub Education?😕
